ros2_node_list:
  ros_control_bridge
  ros2_control_demo_controller
  camera
  robot_visualizer
